.github/workflows/deploy-docs.yml

```
@@ -0,0 +1,44 @@
+        run: cd docs && bun run build
+
+      - name: Setup Pages
+        uses: actions/configure-pages@v4
+
+      - name: Upload artifact
+        uses: actions/upload-pages-artifact@v3
+        with:
+          path: docs/build
+
+      - name: Deploy to GitHub Pages
+        uses: actions/deploy-pages@v4
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)